.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+    root: true,
+    env: {
+        node: true,
+        es6: true
+    },
+    extends: [
+        'eslint:recommended'
+    ],
+    parserOptions: {
+        ecmaVersion: 2015,
```

Comment:
might as well use 2020

Reply:
Sounds good. I wasn't sure if there was much support for es2020 yet but it seems like chrome supports a lot of the new stuff so we should be good👍

Reply:
You can probably throw Babel in there if you need another version in the browser :)